Makefile

```diff
@@ -28,11 +28,11 @@ MANPAGES_MD = $(wildcard docs/*.md)
 
 all: binary docs
 
-binary: skopeo
+binary: skopeo-build
 
 # Build a docker image (skopeobuild) that has everything we need to build.
 # Then do the build and the output (skopeo) should appear in current dir
-skopeo: cmd/skopeo
+skopeo-build: cmd/skopeo
 	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
```

Comment:
What is this actually fixing? “fix dependencies” does not make it obvious what is the failure case.